src/pages/stacks-blockchain/testnet.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Testnet
+description: Learn about testnet phases, timelines, and features
+---
+
+## Introduction
+
+## Testnet phases
+
+- ✅ **Phase 1 (Neon):** is a developer local setup, mono-node, assembling SIP 001, SIP 002, SIP 004 and SIP 005. With this version, developers can not only run Stacks 2.0 on their development machines, but also write, execute, and test smart contracts.
+- ✅ **Phase 2 (Argon):** is the current version of our public testnet. This testnet includes SIP 003, and will be an open-membership public network, where participants will be able to validate and participate in mining testnet blocks.
+- [ ] **Phase 3 (Krypton):** is the upcoming version that we're anticipating to ship in Q3 2020. This version focus on the PoX basics (miner get rewards, Stackers get BTC distributions, etc).
+- [ ] **Phase 4 (Mainnet)** is the fully functional version, that we're intending to ship in Q4 2020.
+
+## Testnet roadmap
+
+✅ Released features&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<input type="checkbox" disabled /> Upcoming
```

Comment:
Rather than duplicate the table here, can we point to the "Testnet phases" section at https://www.blockstack.org/testnet? Or we just want to manually keep the two in sync? (updates are infrequent, so probably fine)

Reply:
I think you are right! I just found a way to link directly to the section on the testnet phase (https://www.blockstack.org/testnet#phases). Will just do that instead!